src/chrome/content/rules/www.2chan.net.xml

```diff
@@ -0,0 +1,14 @@
+<ruleset name="www.2chan.net">
+	<target host="www.2chan.net" />
+	<target host="cgi.2chan.net" />
+	<target host="dat.2chan.net" />
+	<target host="dec.2chan.net" />
+	<target host="may.2chan.net" />
+	<target host="img.2chan.net" />
+	<target host="ipv6.2chan.net" />
+	<target host="jun.2chan.net" />
+	<target host="nov.2chan.net" />
+	<target host="zip.2chan.net" />
+
```

Comment:
Please add <securecookie host=".+" name=".+" />

Reply:
@I-have-50-open-PRs-to-HTTPS-EVERYWHERE 

Since there doesn't seem to be a way to secure the __cfduid and posttime cookie, should I replace:

```xml
<securecookie host=".+" name=".+" />
```

With this?

```xml
<securecookie host="^cgi\.2chan\.net$" name=".+" />
<securecookie host="^dat\.2chan\.net$" name=".+" />
<securecookie host="^dec\.2chan\.net$" name=".+" />
<securecookie host="^img\.2chan\.net$" name=".+" />
<securecookie host="^ipv6\.2chan\.net$" name=".+" />
<securecookie host="^jun\.2chan\.net$" name=".+" />
<securecookie host="^may\.2chan\.net$" name=".+" />
<securecookie host="^nov\.2chan\.net$" name=".+" />
<securecookie host="^zip\.2chan\.net$" name=".+" />
```